tests/api.test.ts

```diff
@@ -228,6 +228,16 @@ describe('api', () => {
         expect(error.code).to.equal('OneSourceSourceMap');
       });
 
+      it('should throw if source map reference column beyond generated last column in line', async function() {
+        try {
+          await explore('data/invalid-map-column.js');
+        } catch (errorResult) {
```

Comment:
Use `assertThrows` or some such. This test will incorrectly pass if `explore` doesn't throw an error.